src/logging.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import logging.handlers
+
+import structlog
+
+from .settings import LogFormat, LoggingSettings
+
+
+def _extract_gunicorn_access_log_event(
+    record: logging.LogRecord,
+    name: str,
+    event_dict: structlog.types.EventDict,
+) -> structlog.types.EventDict:
```

Comment:
Can we add a docstring explaining this logic?

Reply:
Done.